source/common/ssl/context_config_impl.cc

```diff
@@ -66,6 +68,9 @@ ContextConfigImpl::ContextConfigImpl(const envoy::api::v2::CommonTlsContext& con
           tlsVersionFromProto(config.tls_params().tls_maximum_protocol_version(), TLS1_2_VERSION)) {
   // TODO(htuch): Support multiple hashes.
   ASSERT(config.validation_context().verify_certificate_hash().size() <= 1);
+  if (ca_cert_.empty() && !certificate_revocation_list_.empty()) {
+    throw EnvoyException("Cannot have a CRL without a CA certificate");
```

Comment:
Sigh, it looks that this comment got lost somehow.

This should be:

        throw EnvoyException(fmt::format("Failed to load CRL from {} without trusted CA certificates",
                                         config.certificateRevocationListPath()));

to match other errors.

Reply:
@PiotrSikora - Hah, no worries!  Just pushed a commit that changes this and updates the tests.